content/INTRODUCTION.md

```diff
@@ -0,0 +1,25 @@
+**Requirements**: Knowledge in Basic PHP programming, Intermediate PHP programming, and Object-oriented Programming in PHP.
+
+**Duration**:
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+    - Collaborating Around Computing
+    - Recognizing and Defining Computational Problems
+    - Developing and Using Abstractions
+    - Creating Computational Artifacts
+    - Testing and Refining Computational Artifacts
+
+## PHP Advance
+
+Extend and expand your PHP coding skills by learning how to apply the more advanced parts of the server-side scripting language.
```

Comment:
I think you have to rephrase this statement since this is similar to the introduction of the course in other application. As what I have mentioned in my comment, this statement is only for your reference. Please check. Thanks.